weldx/transformations/local_cs.py

```diff
@@ -257,33 +259,21 @@ def _comp_coords():
     @staticmethod
     def _build_orientation(
         orientation: types_orientation,
-        time: pd.DatetimeIndex = None,
-    ):
-        """Create xarray orientation from different formats and time-inputs.
-
-        Parameters
-        ----------
-        orientation :
-            Orientation object or data.
-        time :
-            Valid time index formatted with `_build_time_index`.
-
-        Returns
-        -------
-        xarray.DataArray
-
-        """
+        time: Union[Time, None],
+    ) -> xr.DataArray:
+        """Create xarray orientation from different formats and time-inputs."""
         if orientation is None:
-            orientation = np.array([[1, 0, 0], [0, 1, 0], [0, 0, 1]])
+            orientation = np.eye(3)
+
         if not isinstance(orientation, xr.DataArray):
-            time_orientation = None
             if isinstance(orientation, Rot):
                 orientation = orientation.as_matrix()
             elif not isinstance(orientation, np.ndarray):
                 orientation = np.array(orientation)
 
-            if orientation.ndim == 3:
-                time_orientation = time
+            time_orientation = (
+                time.as_timedelta_index() if orientation.ndim == 3 else None
+            )
```

Comment:
in cases like these the 'old' implementation could be beneficial over the one-liner for seeing the branch coverage in tests

Reply:
Yes, that is actually a good point.